### PR TITLE
Hotcue label fixes

### PR DESCRIPTION
**Pull request by ronso0**

### CODE REVIEW

File: res/skins/Tango/deck_overview.xml [hunk 1]
```diff
@@ -26,6 +26,8 @@ Variables:
     <PlayPosColor>#FF4300</PlayPosColor>
     <PlayedOverlayColor><Variable name="PlayedOverlayColor"/></PlayedOverlayColor>
     <EndOfTrackColor><Variable name="EndOfTrackColor"/></EndOfTrackColor>
+    <LabelTextColor>#ffffff</LabelTextColor>
+    <LabelBackgroundColor>#85000000</LabelBackgroundColor>
```
Comment (Be-ing) [inline res/skins/Tango/deck_overview.xml:30]: I'm not sure I like this. The waveform underneath is not as visible.

Comment (ronso0) [reply res/skins/Tango/deck_overview.xml:30]: IMO the default opacity of the label's box is way to low.
The labels should add information not noise..
As you can see: as soon as a loop region comes into play the labels are difficult to read.

this PR:
image

default opacity:
image

Deere default:
image

Comment (Be-ing) [reply res/skins/Tango/deck_overview.xml:30]: > as soon as a loop region comes into play the labels are difficult to read

You're right; that was not a use case I considered.

Comment (ronso0) [reply res/skins/Tango/deck_overview.xml:30]: ok. I didn't check this in the other skins. I'll do so and supply another PR later

**Pull request merged**